sources/packages/backend/apps/api/src/route-controllers/institution-locations/institution-location.institutions.controller.ts

```diff
@@ -211,6 +212,9 @@ export class InstitutionLocationInstitutionsController extends BaseController {
       ...transformAddressDetailsForAddressBlockForm(
         institutionLocation.data.address,
       ),
+      isBCPublicInstitution:
+        institutionLocation.institution.institutionType.id ===
+        INSTITUTION_TYPE_BC_PUBLIC,
     };
```

Comment:
Maybe not a part of this PR but this code (entire return) can be moved to `institution-location.controller.service`.

Reply:
This code is not necessary unless I am missing something. Please see the comment https://github.com/bcgov/SIMS/pull/3734/files#r1774067185